### PR TITLE
Merge QDMEwald

### PR DESCRIPTION
**Pull request by junghans**
To Do:
- [x] add comment to `NOTICE` file
- [x] add copyright header to new files

NOT To Do (in this PR):
- [x] fix CMake
- [x] integrate with CI

### CODE REVIEW

File: src/QDMEwald/src/QDMEwald.hpp [hunk 1]
```diff
@@ -0,0 +1,204 @@
+/*
+ *            Copyright 2019 Juelich Supercomputing Centre
```
Comment (junghans) [inline src/QDMEwald/src/QDMEwald.hpp:2]: @rhalver is that what you need?

File: src/QDMEwald/tests/test_kspace.cpp [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Juelich Supercomputing Centre
```
Comment (junghans) [inline src/QDMEwald/tests/test_kspace.cpp:2]: @JensWehner the tests already had the VOTCA header !? But I added JSC, ok?

Comment (JensWehner) [reply src/QDMEwald/tests/test_kspace.cpp:2]: Sure.

File: src/QDMEwald/src/QDMEwald.hpp (hunk 1 above)
Comment (rhalver) [reply src/QDMEwald/src/QDMEwald.hpp:2]: Copyright 2019 Forschungszentrum Juelich GmbH, Germany

It needs to mention the research center itself, not the institute, usually we mention the authors themselves  in it, but I am not sure if it is required.

File: NOTICE [hunk 3]
```diff
@@ -3,11 +3,11 @@ Copyright 2009-2020 The VOTCA Development Team
...
+and at Forschungszentrum Juelich GmbH, Germany.
 
 This software contains code, which was
 derived from VOTCA KMC, MOO and CTP project.
 
 The implementation of the Ewald method is contributed
-by Rene Halver, Godehard Sutmann from Juelich Supercomputing
-Centre.
+by Rene Halver, Godehard Sutmann from Forschungszentrum 
+Juelich GmbH, Germany.
```
Comment (baumeier) [inline NOTICE:13]: @rhalver How about this mention in the NOTICE file?

Comment (rhalver) [reply NOTICE:13]: I think that should be fine, I will change the copyrights in the source files and will update them.

Comment (rhalver) [reply NOTICE:13]: It seems I am not allowed to push the changes to the branch? Errors out with 403.

Comment (baumeier) [reply NOTICE:13]: I added you to the xtp group. How about now?

Comment (rhalver) [reply NOTICE:13]: Just pushed the changes, thanks.

**Comment by junghans** [comment]
@baumeier @rhalver please approve.

**Pull request merged**